### ISSUE TITLE
Feature request: Bulk import of stat blocks

### ISSUE THREAD

**Issue by baileymh** [Lavaeolous/PF1-StatBlock-Converter-Module#387]
Really appreciate all your hard work. 

It would be very helpful for people creating AP modules, or who are converting material to be able to do bulk imports. 

I recognize this could be tricky or slightly more prone to error. So I'm not sure how difficult it might be to code. 

In my conception of it, it could be a single text file with the stat blocks listed, separated by a series of characters that wouldn't normally occur in the stat block, (e.g., "========") . 
It likely wouldn't be able to show the parser warnings or errors, but the time saved would, in my opinion, outweigh minor mistakes that could be fixed by manually re-importing and adjusting. 

I know you're working on spell stuff, so no rush on this.

**Comment by Lavaeolous** [comment]
Yeah, that would be nice. I'll keep it in mind but do not see it in the 3.x iteration of sbc.

**Comment by baileymh** [comment]
Completely fair of it not being included in the near future. Thanks!
